udata/core/dataset/rdf.py

```diff
@@ -395,7 +406,7 @@ def dataset_from_rdf(graph, dataset=None, node=None):
         dataset.acronym = acronym
 
     tags = [tag.toPython() for tag in d.objects(DCAT.keyword)]
-    tags += [theme.toPython() for theme in d.objects(DCAT.theme)]
+    tags += [theme.toPython() for theme in d.objects(DCAT.theme) if not isinstance(theme, RdfResource)]
```

Comment:
Avoid using `RdfResource` to populate tags (they won't make sense anyway as they would mostly be links).